ros/api/v1/call_to_action.py

```diff
@@ -1,9 +1,9 @@
 from flask_restful import Resource
 from flask import request
 from ros.lib.utils import (
-    org_id_from_identity_header, system_ids_by_org_id)
+    org_id_from_identity_header, systems_with_profile)
```

Comment:
@r14chandra - Any specific reason that you extracted query part into a separate method?

Reply:
We can use this method in the PR for /is_configured - #254. Once this PR is in, I'll make the required changes in that PR.

Reply:
No. of systems having performance profile is used in some place like under `is_configured` and `executive_report`